services/logger.js

```diff
@@ -1,16 +1,32 @@
 const chalk = require('chalk');
 
-function logLine(color, message) {
-  console.log(`${chalk[color]('>')} ${message}`);
-}
+class Logger {
+  constructor(silent) {
+    this.silent = silent;
+  }
+
+  _logMessage(message, force) {
```

Comment:
[Linter] Do you really need to bypass the `"no-underscore-dangle"` linter rule ? Can't you just use `logMessage` or anything else which wouldn't require a linter exception ?